### PR TITLE
[FIX] fix message prompt

### PR DESCRIPTION
**Pull request by lgharib**
11 lmg fix message prompt

### CODE REVIEW

File: project_event/models/project_task.py [hunk 1]
```diff
@@ -1009,6 +1050,22 @@ def is_hr_resource_booked(self, attendee):
         ])
         return len(overlaps_partners) > 0
 
+    def is_hr_resource_double_booked(self, attendee, date_start, date_end):
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: For the method is_hr_resource_booked, i forget to check if it is not the same event_task_id, so when added we have to merge the 2 methods and keep only one

File: project_resource_calendar/models/calendar_event.py [hunk 2]
```diff
@@ -371,3 +352,54 @@ def unlink(self):
...
         return super(CalendarEvent, self).unlink()
+
+    def is_hr_resource_double_booked(self, attendee,
+                                     date_start=None, date_end=None):
+        if not date_end and not date_start:
+            date_start = self.date_start
+            date_end = self.date_end
+
+        overlaps_partners = self.env['calendar.event'].search([
+            ('id', '!=', self.id),
```
Comment (rimbendhaou) [inline project_resource_calendar/models/calendar_event.py:363]: we need to factor this redandant code, we can use a constatnt (OVERLAP_DOMAIN = [
            ('id', '!=', self.id),('start', '<', date_end),
            ('stop', '>', date_start),
            ('state', '!=', 'cancelled'),] and use it each time (we simply add the specific fields to check ( ('partner_ids', 'in', attendee.id))

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_task.py [hunk 3]
```diff
@@ -727,6 +727,50 @@ def get_booked_resources(self):
                             '<br>'
         return res
 
+    def get_double_booked_resources(self, date_start=None, date_end=None):
```
Comment (eilst) [inline project_event/models/project_task.py:730]: hey, this methods are duplicated code. Just a comment, but you could reference the object and without creating it and make these verifications.

File: project_resource_calendar/models/calendar_event.py [hunk 4]
```diff
@@ -371,3 +352,50 @@ def unlink(self):
...
+            ('partner_ids', 'in', attendee.id),
+            ('start', '<', date_end),
+            ('stop', '>', date_start),
+            ('state', '!=', 'cancelled'),
+        ])
+        return len(overlaps_partners) > 0
+
+    def get_double_booked_resources(self, date_start=None, date_end=None):
+        booked_resources = []
+        overlap_domain = [
```
Comment (rimbendhaou) [inline project_resource_calendar/models/calendar_event.py]: overlap_domain should be after (if not date_end and not date_start:)

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**